src/vs/workbench/services/contextmenu/electron-sandbox/contextmenuService.ts

```diff
@@ -97,10 +97,20 @@ class NativeContextMenuService extends Disposable implements IContextMenuService
 			let x: number;
 			let y: number;
 
-			const zoom = getZoomFactor();
+			let zoom = getZoomFactor();
 			if (dom.isHTMLElement(anchor)) {
 				const elementPosition = dom.getDomNodePagePosition(anchor);
 
+				let testElement: HTMLElement | null = anchor;
```

Comment:
This raises a lot of questions:
- is this for all platforms or only windows?
- what does it actually do, maybe a comment would help?

Reply:
Added a comment to explain, it applies to all platforms. You might recall a few weeks ago that I had to fix an issue where the layout control menu was drawn in the wrong position when zoom was applied on macOS. The fix that I made is incompatible with the required css changes to support the native controls. So, this is essentially a new fix for that problem that is required with adoption of the WCO